assignment/core/src/main/java/com/assignment/core/services/impl/EMICalculationServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.assignment.core.services.impl;
+
+import com.assignment.core.services.EMICalculationService;
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+        service = EMICalculationService.class
```

Comment:
should give name and immediate property in services

Reply:
Is it necessary to add immediate property if service is working as I have used it inside servlet ?